test/functional/rpc_net.py

```diff
@@ -319,7 +319,9 @@ def test_getnodeaddresses(self):
 
     def test_addpeeraddress(self):
         self.log.info("Test addpeeraddress")
-        self.restart_node(1, ["-checkaddrman=1", "-test=addrman"])
+        # The node has an existing, non-deterministic addrman from a previous test.
```

Comment:
Which is that previous test?

Reply:
Any previous test that started the node without a deterministic addrman. I don't think it's needed in the comment.